issuer/src/main.rs

```diff
@@ -1003,11 +1034,36 @@ fn verify_principal_owns_credential(
 ) -> Result<(), IssueCredentialError> {
     let mut group_name = group_name(&credential_spec.credential_type)?;
     if credential_spec.credential_type == "VerifiedMember" {
-        group_name = get_string_arg_value("groupName", &credential_spec)
+        group_name = get_string_arg_value("groupName", credential_spec)
             .map_err(IssueCredentialError::UnauthorizedSubject)?;
     }
     if let Some(group_record) = groups.get(&(group_name.clone(), owner).into()) {
         if let Some(member_record) = group_record.members.get(&user) {
+            let stored_spec: CredentialSpec = OrdCredentialSpec {
+                credential_type: credential_spec.credential_type.clone(),
+                arguments: member_record.vc_arguments.clone(),
+            }
+            .into();
+            if credential_spec.credential_type == "VerifiedAge" {
```

Comment:
Could we use a variable for `"VerifiedAge"` here and in the GROUP_TYPE?

Reply:
You mean a constant, right?  Done.